pkg/loadbalancerservice/controller.go

```diff
@@ -235,13 +236,20 @@ func (c *LoadbalancerServiceController) updateDefaultRouterServiceStatus(ips []s
 		if err != nil {
 			return false, err
 		}
-		klog.Infof("Updating default router service status: %v", ips)
 		newStatus := &corev1.LoadBalancerStatus{}
 		for _, ip := range ips {
 			newStatus.Ingress = append(newStatus.Ingress, corev1.LoadBalancerIngress{
 				IP: ip,
 			})
 		}
+
+		equal := slices.EqualFunc(svc.Status.LoadBalancer.Ingress, newStatus.Ingress, func(oldIP, newIP corev1.LoadBalancerIngress) bool {
```

Comment:
Should we sort the slices before comparing? Are slices with the same elements considered equal or order makes a difference?

Reply:
They are already sorted here: https://github.com/openshift/microshift/blob/main/pkg/loadbalancerservice/router.go#L119
This was done in a previous PR.